runtime/doc/options.txt

```diff
@@ -6342,9 +6342,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 'term'			string	(default is $TERM, if that fails:
 				      in the GUI: "builtin_gui"
 					  on Mac: "mac-ansi"
-				       on MS-DOS: "pcterm"
```

Comment:
There are some more refs to pcterm:
```diff
diff --git a/runtime/doc/options.txt b/runtime/doc/options.txt
index 1a9c951..79db9b1 100644
--- a/runtime/doc/options.txt
+++ b/runtime/doc/options.txt
@@ -779,7 +779,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	Normally this option would be set in the vimrc file.  Possibly
 	depending on the terminal name.  Example: >
-		:if &term == "pcterm"
+		:if &term == "xterm"
 		:  set background=dark
 		:endif
 <	When this option is set, the default settings for the highlight groups
@@ -7099,6 +7099,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 			global
 	The number of microseconds to wait for each character sent to the
 	screen.  When non-zero, characters are sent to the terminal one by
-	one.  For MS-DOS pcterm this does not work.  For debugging purposes.
+	one.  For debugging purposes.
 
  vim:tw=78:ts=8:ft=help:noet:norl:
diff --git a/runtime/doc/term.txt b/runtime/doc/term.txt
index 961ec1b..9c74a7e 100644
--- a/runtime/doc/term.txt
+++ b/runtime/doc/term.txt
@@ -425,11 +425,9 @@ cleared when scrolling).
 Unfortunately it is not possible to deduce from the termcap how cursor
 positioning should be done when using a scrolling region: Relative to the
 beginning of the screen or relative to the beginning of the scrolling region.
-Most terminals use the first method.  A known exception is the MS-DOS console
-(pcterm).  The 't_CS' option should be set to any string when cursor
-positioning is relative to the start of the scrolling region.  It should be
-set to an empty string otherwise.  It defaults to "yes" when 'term' is
-"pcterm".
+Most terminals use the first method.  The 't_CS' option should be set to any
+string when cursor positioning is relative to the start of the scrolling
+region.  It should be set to an empty string otherwise.
 
 Note for xterm users: The shifted cursor keys normally don't work.  You can
 	make them work with the xmodmap command and some mappings in Vim.
```